app/controllers/services_and_information_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ServicesAndInformationController < ApplicationController
+  def index
+    base_path = "/government/organisations/#{params[:organisation_id]}/services-information"
+    content_item = Services.content_store.content_item!(base_path).to_hash
+    @navigation_helpers = GovukNavigationHelpers::NavigationHelper.new(content_item)
+    @organisation = content_item.dig("links", "parent", 0)
+    @grouped_links = parsed_grouped_links
+    set_slimmer_analytics_headers
+  end
+
+private
+
+  def set_slimmer_analytics_headers
+    set_slimmer_headers(organisations: "<#{@organisation['analytics_identifier']}>")
```

Comment:
Yup, I copied over the functionality but I'll raise a separate PR to update this.